docs/dev/table/sql/gettingStarted.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Getting Started - Flink SQL"
+nav-parent_id: sql
+nav-pos: 0
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+* This will be replaced by the TOC
+{:toc}
+
+Flink SQL enables SQL developers to design and develop the batch or streaming application without writing the Java, Scala, or any other programming language code. It provides a unified API for both batch and streaming APIs. As a user, you can perform powerful transformations. Flink’s SQL support is based on [Apache Calcite](https://calcite.apache.org/) which implements the SQL standard.
+
+In addition to the SQL API, Flink also has Table API as well with similar semantics as SQL. Table API is a Language integrated API, where we use a specific programming language to write the queries or call the API. For example, we create the table environment, get a table object, and apply different methods that return table API objects. It supports different languages e.g. Java, Scala, Python. 
+ 
+Flink SQL and Table API are just two ways to write queries that use the same API underneath. It wouldn’t be wrong if we say Table API is a wrapper on top of the streaming API. SQL API is a more descriptive way of writing queries using well-known SQL standards where we usually have ‘select * from Table’ pattern.  Table API query starts with from clause, followed by joins and where clause, and then finally projection or select at the last. SQL API is easy to learn and almost everyone knows it already. All the queries are optimized for efficient execution. We will focus on the Flink SQL API, while you can read more about Table API [here]({{ site.baseurl }}/dev/table/).
+
+### Pre-requisites
+You only need to have basic knowledge of SQL to follow along. You will not need to write Java or Scala code or use an IDE.
+
+### Installation
+There are various ways to install the Flink. You can download the source code, compile it, and run it. Another option is to have it running inside the container. Probably the easiest one is to download the binaries and run it locally for experimentation. We assume local installation for the rest of the tutorial. You can start a local cluster using the following command from the installation folder
```

Comment:
> There are various ways to install the Flink

Its not *the flink* just Flink. "There are various ways to install Flink". 

> You can download the source code, compile it, and run it.

Compiling flink is difficult, I'd rather not mention that in a getting started guide. 

If we are guiding users towards local installation, how about a link to the downloads page.